examples/notebooks/6_Noise_Simulation.ipynb

```diff
@@ -673,6 +808,205 @@
     "gad.to_other_language(Language.QISKIT)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "SuperOp([[0.908     +0.j, 0.        +0.j, 0.        +0.j, 0.138     +0.j],\n",
+       "         [0.        +0.j, 0.87749644+0.j, 0.        +0.j, 0.        +0.j],\n",
+       "         [0.        +0.j, 0.        +0.j, 0.87749644+0.j, 0.        +0.j],\n",
+       "         [0.092     +0.j, 0.        +0.j, 0.        +0.j, 0.862     +0.j]],\n",
+       "        input_dims=(2,), output_dims=(2,))"
+      ]
+     },
+     "execution_count": 29,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "## QuantumChannel representation\n",
+    "gad.to_other_language(Language.QISKIT).to_quantumchannel()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4- PhaseDamping noise model\n",
+    "\n",
+    "We can define the ``PhaseDamping`` noise model by providing the decoherence rate ``gamma``, which represents the probability of phase error occurring during the quantum operation. ``PhaseDamping`` noise model introduces phase errors into a quantum system caused by interactions with the environment, leading to decoherence but leaving the energy of the system unchanged.\n",
+    "\n",
+    "Similar to other noise models, One can target all qubits of a circuit or just select specific ones. If no targets are specified, the noise model will be applied to all qubits in the circuit. By default, the parameter gates is an empty list."
+   ]
```

Comment:
More precisely, by default the parameter gates is None, and then is turned into an empty list, but not sure those details are important from a user point of view

Reply:
Will remove the info about the gates from it? what do you think?

Reply:
you can maybe just say that if we don't fill the parameter 'gates', then all gates in the circuit are affected by the noise